### PR TITLE
Fix and test transformation plan generation

### PR DESCRIPTION
**Pull request by nyuuyn**
#### Short Description
Adds some fixes within the transformation plan builder and multiple tests for the plan generators

### CODE REVIEW

File: org.opentosca.planbuilder/org.opentosca.planbuilder.core.bpel/src/main/java/org/opentosca/planbuilder/core/bpel/typebasedplanbuilder/BPELTransformationProcessBuilder.java [hunk 1]
```diff
@@ -425,6 +419,15 @@ public BPELPlan buildPlan(String sourceCsarName, AbstractDefinitions sourceDefin
             transformationBPELPlan.getBpelMainSequenceOutputAssignElement(),
             "CREATED", targetServiceInstanceURL);
 
+        String planInstanceUrlVarName = this.serviceInstanceHandler.findPlanInstanceUrlVariableName(transformationBPELPlan);
+        this.serviceInstanceHandler.appendSetServiceInstanceState(transformationBPELPlan,
+            transformationBPELPlan.getBpelMainFlowElement(),
+            "RUNNING", planInstanceUrlVarName);
```
Comment (wederbn) [inline org.opentosca.planbuilder/org.opentosca.planbuilder.core.bpel/src/main/java/org/opentosca/planbuilder/core/bpel/typebasedplanbuilder/BPELTransformationProcessBuilder.java:425]: PlanInstanceState.RUNNING?

File: org.opentosca.planbuilder/org.opentosca.planbuilder.core.bpel/src/main/java/org/opentosca/planbuilder/core/bpel/typebasedplanbuilder/BPELTransformationProcessBuilder.java [hunk 2]
```diff
@@ -425,6 +419,15 @@ public BPELPlan buildPlan(String sourceCsarName, AbstractDefinitions sourceDefin
...
+
+        this.serviceInstanceHandler.appendSetServiceInstanceState(transformationBPELPlan,
+            transformationBPELPlan.getBpelMainSequenceOutputAssignElement(),
+            "FINISHED", planInstanceUrlVarName);
```
Comment (wederbn) [inline org.opentosca.planbuilder/org.opentosca.planbuilder.core.bpel/src/main/java/org/opentosca/planbuilder/core/bpel/typebasedplanbuilder/BPELTransformationProcessBuilder.java:429]: PlanInstanceState.FINISHED?

File: test.properties [hunk 3]
```diff
@@ -40,3 +40,4 @@ org.opentosca.container.collaboration.ports=
 org.opentosca.deployment.tests=false
 org.opentosca.bus.management.mocking=false
 org.opentosca.engine.ia.keepfiles=true
+org.opentosca.test.local.repository.path=/tmp/tosca-definitions-public
```
Comment (wederbn) [inline test.properties:43]: Don`t we need org.opentosca.test.remote.repository.url in the test.properties too?

**Review by wederbn — APPROVED** [review]
LGTM

File: test.properties (hunk 3 above)
Comment (nyuuyn) [reply test.properties:43]: Only if the test cases clone the test repository themselves which is not the case

**Review by zimmerml — APPROVED** [review]

**Pull request merged**